src/common/data/routes.ts

```diff
@@ -5,3 +5,6 @@ export const ORDER_PAGE_ROUTE = "/comnenzi";
 export const LOCATIONS_PAGE_ROUTE = "/locatii";
 export const CONTACT_PAGE_ROUTE = "/contact";
 export const CART_PAGE_ROUTE = "/cart";
+export const CATEGORY_PIZZA_ROUTE= "/categorie/pizza"
```

Comment:
Please extract the "/categorie" in a constant called CATEGORY_DEFAULT_ROUTE and use it in each subroute

Reply:
Extracted default route